### PR TITLE
Zigbee DALI Device

### PR DESCRIPTION
**Pull request by Danit2**
- Small correction on the name
- Make it possible to controll two light when two DALI devices are connectet
- Delete OTA because there is no Firmware online

### CODE REVIEW

File: devices/osram.js [hunk 1]
```diff
@@ -382,11 +382,17 @@ module.exports = [
...
     {
-        zigbeeModel: ['Zigbee 3.0 DALI CONV LI'],
+        zigbeeModel: ['Zigbee 3.0 DALI CONV LI\u0000'],
         model: '4062172044776',
         vendor: 'OSRAM',
         description: 'OSRAM Zigbee 3.0 DALI CONV LI dimmer for DALI-based luminaires',
-        extend: extend.ledvance.light_onoff_brightness(),
-        ota: ota.ledvance,
+        fromZigbee: [fz.on_off, fz.brightness, fz.level_config, fz.power_on_behavior, fz.ignore_basic_report],
+        toZigbee: [tz.light_onoff_brightness, tz.ignore_transition, tz.ignore_rate, tz.light_brightness_move,
```
[the remark this answers is not in the document]
Comment (Koenkk) [reply devices/osram.js]: Could you provide the debug log when starting z2m? 

See https://www.zigbee2mqtt.io/information/debug.html on how to enable debug logging.

[the remark this answers is not in the document]
Comment (Danit2) [reply devices/osram.js]: [log.txt](https://github.com/Koenkk/zigbee-herdsman-converters/files/7445895/log.txt)

Here the file. 
Is this ok or you need more? 
thanks

[the remark this answers is not in the document]
Comment (Koenkk) [reply devices/osram.js]: I've checked locally but this config works OK, in your situation the definition is indeed not loaded correctly somehow.

- Make sure you updated the correct file and that there are no external converters.
- Try removing `\u0000` from the `zigbeeModel`.

[the remark this answers is not in the document]
Comment (Danit2) [reply devices/osram.js]: OK, you can merge this. 

Thanks for your help.

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**